enigma/src/main/java/cuchaz/enigma/source/Decompiler.java

```diff
@@ -1,5 +1,12 @@
-package cuchaz.enigma.source;
-
-public interface Decompiler {
-    Source getSource(String className);
-}
+package cuchaz.enigma.source;
+
+import cuchaz.enigma.translation.mapping.EntryRemapper;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public interface Decompiler {
+    default Source getSource(String className) {
+    	return getSource(className, null);
+	}

```

Comment:
Indentation is weird here

Reply:
Fixed in https://github.com/FabricMC/Enigma/pull/336/commits/ad1d75cf4a17236d48ce1a38f25057f388300068